README.md

```diff
@@ -3,13 +3,13 @@
 ## Definitions
 Define the following: 
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
-- Activation Record/Call
-- Infinite Recursion/Stack Overflow/Stack too deep
-- Tail Recursion
+- Recursion: When a function/method calls itself
+- Recursive Case: input for which the program calls itself
+- Base Case: input for which the program produces an answer without recurring
+- Activation Chain/Stack: the chain of iterations and base case. (the chain of bubbles we used to solve the recursive methods) 
+- Activation Record/Call: Each bubble within the chain.
+- Infinite Recursion/Stack Overflow/Stack too deep: when no base case can be reached.
+- Tail Recursion: fast type of recursion which does not need to go through the activation chain.
```

Comment:
This is not totally true - I don't think. But we didn't cover this in class. We may in the future, but it's not important to know in this moment.